src/main/java/arts/Arts.java

```diff
@@ -0,0 +1,131 @@
+package arts;
+
+import arts.util.Parser;
+import arts.util.Storage;
+import arts.util.Ui;
+import arts.task.TaskList;
+import arts.enums.CommandType;
+import arts.command.AddDeadlineCommand;
+import arts.command.AddEventCommand;
+import arts.command.MarkCommand;
+import arts.command.UnmarkCommand;
+import arts.command.DeleteCommand;
+import arts.command.AddTodoCommand;
+import arts.command.FindCommand;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Arts class is the main entry point for the Arts application.
+ * It manages the initialization and execution of the application, handling user commands
+ * and coordinating between different components like storage, UI, and task management.
+ */
+public class Arts {
+    private static final DateTimeFormatter[] INPUT_FORMATTERS = new DateTimeFormatter[]{
+            DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm"),
+            DateTimeFormatter.ofPattern("d/M/yyyy HHmm")
+    };
+
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
+    private final Parser parser;
+
+    /**
+     * Constructs an Arts object with the specified file path for task storage.
+     * Initializes the UI, storage, parser, and loads existing tasks.
+     *
+     * @param filePath The path of the file where tasks are stored.
+     */
+    public Arts(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        parser = new Parser();
+        TaskList tempTasks;
+        try {
+            tempTasks = new TaskList(storage.load());
+        } catch (ArtsException e) {
+            ui.showError(e.getMessage());
+            tempTasks = new TaskList();
+        }
+        tasks = tempTasks;
+    }
+
+    /**
+     * Runs the Arts application, processing user commands in a loop until exit.
+     */
+    public void run() {
+        ui.showWelcome();
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String input = ui.readCommand();
+                CommandType command = parser.parseCommand(input);
+                String[] parts = parser.parseArguments(input);
+
+                switch (command) {
+                    case BYE:
```

Comment:
There should be no indentation for case clauses according to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html) mentioned.

You can tweak your Intellij IDE to ensure future switch case follow no indentation [here](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-switch-case-style).